services/QuillLMS/client/app/bundles/Staff/components/evidence/configureRules/ruleViewForm.tsx

```diff
@@ -268,7 +270,7 @@ const RuleViewForm = ({
           rulePromptsDisabled={rulePromptsDisabled}
           setRulePrompts={setRulePrompts}
         />}
-        {(isUniversal || isSemantic) && <RuleUniversalAttributes
+        {(isUniversal || isSemantic || isLowConfidence) && <RuleUniversalAttributes
```

Comment:
I feel like we can probably pull this out into a const, something like `showUniversalAttributes`